package.json

```diff
@@ -41,59 +41,59 @@
   "dependencies": {
     "bootstrap-sass": "3.4.1",
     "code-prettify": "^0.1.0",
-    "datatables.net": "~1.13.8",
-    "dompurify": "^2.4.4",
-    "es5-shim": "2.3.0",
+    "datatables.net": "~1.13.11",
+    "dompurify": "^2.5.7",
+    "es5-shim": "4.6.7",
     "fast-json-patch": "github:wet-boew/JSON-Patch#wb1.0+1.1.4",
     "html5shiv": "^3.7.3",
     "jquery": "2.2.4",
-    "jquery-validation": "1.19.5",
+    "jquery-validation": "1.21.0",
     "jsonpointer.js": "^0.4.0",
-    "magnific-popup": "git+https://github.com/wet-boew/Magnific-Popup.git#1.0.0+keyboard_trap_fix",
-    "mathjax": "^3.2.0",
-    "proj4": "2.3.3",
+    "magnific-popup": "github:wet-boew/Magnific-Popup#1.0.0+keyboard_trap_fix",
+    "mathjax": "^3.2.2",
+    "proj4": "2.12.1",
```

Comment:
That one is active, they updated to the version 2.14 yesterday - No need to update if it does not impact geomap